### PR TITLE
Integrate Ant Design

### PR DESCRIPTION
**Pull request by agusit7**
#### :link: Trello board reference:

- [Task de integración de framework css](https://trello.com/c/Ai2GPpZZ)

#### :pencil2: Description:

Integrates Ant Design ready for theming, adds webpack and babel configurations.
The launcher button color was modified just for testing purposes, it will be corrected in the future styling task.

---

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,11 +61,12 @@
     "html-loader": "^0.5.5",
     "html-webpack-plugin": "^3.2.0",
     "husky": "^3.0.4",
+    "less": "^3.10.3",
+    "less-loader": "^5.0.0",
     "lint-staged": "^8.0.4",
     "mini-css-extract-plugin": "^0.8.0",
     "node-sass": "^4.12.0",
```
Comment (Tintef) [inline package.json]: You should remove this one too 🤔

File: theme/theme.less [hunk 2]
```diff
@@ -0,0 +1 @@
+@primary-color: #ffa500;
```
Comment (Tintef) [inline theme/theme.less]: Don't forget to use our color palette! (just a reminder)

File: package.json (hunk 1 above)
Comment (agusit7) [reply package.json]: You're right! Nice catch!

File: theme/theme.less (hunk 2 above)
Comment (agusit7) [reply theme/theme.less]: Yep, thanks! It will be added when the colors get implemented.

**Pull request merged**